coding-guides/understanding-raw-events.md

```diff
@@ -0,0 +1,111 @@
+# Raw Events
+
+Raw events are used for uncached messages. How would we use raw events? We can use them to check when a message has been edited, reaction was added or removed etc. So, with that being said, let's get started!
```

Comment:
Specify that raw events can do more than just work with messages, but you are using message management as an example

Reply:
or better yet, specify that raw events trigger on all events and all data from discord, and you are using it *specifically* because reactionAdd doesn't trigger on uncached messages